src/napari_imagej/widgets/menu.py

```diff
@@ -165,7 +165,10 @@ def send_chosen_layer(self):
             layer = choices["layer"]
             if isinstance(layer, Layer):
                 # Pass the relevant data to ImageJ2
-                ij().ui().show(ij().py.to_java(layer))
+                self._show(layer)
+
+    def _show(self, layer):
+        ij().thread().queue(lambda: ij().ui().show(ij().py.to_java(layer)))
```

Comment:
What if we instead made a function that encapsulated the `ij().thread().queue()` behavior? Something like (forgive the naming, I'm not too familiar with java gui terms)
```python
def _java_gui_execute(self, func: Callable[[], None}):
  ij().thread().queue(callable)
```
Do you think it's worth it?

Reply:
I dunno, it only saves a couple of characters. If you prefer: I've been meaning to add convenience functions to scyjava for `EventQueue.invokeLater` and `EventQueue.invokeAndWait`. Then you could use those, and we would not need to call the thread service. The downside is that if anyone is using an ImageJ2 with customized thread service, we would not be respecting their overrides of those behaviors.

Reply:
I'd prefer that, but I don't think we want to hold up this PR to wait. Maybe a TODO comment is enough...

Reply:
TODOs added in d5218f5